coderd/database/dump/main.go

```diff
@@ -31,6 +31,11 @@ func main() {
 	}
 
 	cmd := exec.Command(
+		"docker",
+		"run",
+		"--rm",
+		"--network=host",
+		"postgres:13",
```

Comment:
This was a bit out of scope, but makes everything use Docker, which makes changing our versions simpler!